getting-started/setup-building.rst

```diff
@@ -655,3 +655,52 @@ every rule.
 
 
 .. _issue tracker: https://github.com/python/cpython/issues
+
+
+.. _using-codespaces:
+
+Contribute Using GitHub Codespaces
```

Comment:
According to the devguide docs style guide, headings should be written using "sentence case":

https://devguide.python.org/documentation/style-guide/#capitalization

> In the Python documentation, the use of sentence case in section titles is preferable, but consistency within a unit is more important than following this rule. If you add a section to a chapter where most sections are in title case, you can either convert all titles to sentence case or use the dominant style in the new section title.

Reply:
```suggestion
Contribute using GitHub Codespaces
```

Reply:
Thanks for pointing that out! I'll go through and adjust the titles to match!

Reply:
Actually, thanks @pradyunsg for the suggestions to do just that 😄

Reply:
Thanks! There's a lot of guidelines; it's hard to remember all of them. I misstep all the time 😄